[2021-12-28 13:47:47,419] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_galway.scrap_data scheduled__2021-12-27T09:35:00+00:00 [queued]>
[2021-12-28 13:47:47,445] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_galway.scrap_data scheduled__2021-12-27T09:35:00+00:00 [queued]>
[2021-12-28 13:47:47,446] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:47:47,447] {taskinstance.py:1242} INFO - Starting attempt 4 of 4
[2021-12-28 13:47:47,448] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:47:47,471] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-27 09:35:00+00:00
[2021-12-28 13:47:47,485] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_galway', 'scrap_data', 'scheduled__2021-12-27T09:35:00+00:00', '--job-id', '5113', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_galway.py', '--cfg-path', '/tmp/tmpzruzbzfy', '--error-file', '/tmp/tmpfzne_to3']
[2021-12-28 13:47:47,487] {standard_task_runner.py:77} INFO - Job 5113: Subtask scrap_data
[2021-12-28 13:47:47,481] {standard_task_runner.py:52} INFO - Started process 55896 to run task
[2021-12-28 13:47:47,592] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_galway.scrap_data scheduled__2021-12-27T09:35:00+00:00 [running]> on host 0c125a5bba95
[2021-12-28 13:47:47,804] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_galway
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T09:35:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T09:35:00+00:00
[2021-12-28 13:47:47,841] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 13:47:47,846] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py galway
[2021-12-28 13:47:49,404] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 13:47:49,505] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 13:47:49,506] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 13:47:49,507] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 13:47:49,508] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 13:47:49,508] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 13:47:49,557] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 13:47:49,558] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 13:47:49,559] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 13:47:49,560] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 13:47:49,560] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 13:47:49,561] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 13:47:49,562] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 13:47:49,563] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 13:47:49,563] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 13:47:49,564] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 13:47:49,565] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 13:47:49,566] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 13:47:49,566] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 13:47:49,567] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 13:47:49,568] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 13:47:49,568] {spark_submit.py:514} INFO - files                   null
[2021-12-28 13:47:49,569] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 13:47:49,569] {spark_submit.py:514} INFO - archives                null
[2021-12-28 13:47:49,570] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 13:47:49,571] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:47:49,571] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-28 13:47:49,572] {spark_submit.py:514} INFO - childArgs               [galway]
[2021-12-28 13:47:49,572] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 13:47:49,573] {spark_submit.py:514} INFO - packages                null
[2021-12-28 13:47:49,574] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 13:47:49,574] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 13:47:49,575] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 13:47:49,576] {spark_submit.py:514} INFO - 
[2021-12-28 13:47:49,576] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 13:47:49,577] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 13:47:49,578] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:47:49,578] {spark_submit.py:514} INFO - 
[2021-12-28 13:47:49,579] {spark_submit.py:514} INFO - 
[2021-12-28 13:47:49,878] {spark_submit.py:514} INFO - Main class:
[2021-12-28 13:47:49,880] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 13:47:49,881] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 13:47:49,882] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:47:49,883] {spark_submit.py:514} INFO - null
[2021-12-28 13:47:49,884] {spark_submit.py:514} INFO - galway
[2021-12-28 13:47:49,885] {spark_submit.py:514} INFO - --verbose
[2021-12-28 13:47:49,886] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 13:47:49,888] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-28 13:47:49,889] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:47:49,890] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 13:47:49,891] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 13:47:49,892] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 13:47:49,893] {spark_submit.py:514} INFO - 
[2021-12-28 13:47:49,895] {spark_submit.py:514} INFO - 
[2021-12-28 13:47:49,896] {spark_submit.py:514} INFO - 
[2021-12-28 13:47:50,920] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 13:47:50,938] {spark_submit.py:514} INFO - 21/12/28 13:47:50 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 13:47:51,042] {spark_submit.py:514} INFO - 21/12/28 13:47:51 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 13:47:51,243] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO ResourceUtils: ==============================================================
[2021-12-28 13:47:51,245] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 13:47:51,246] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO ResourceUtils: ==============================================================
[2021-12-28 13:47:51,247] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-28 13:47:51,289] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 13:47:51,310] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 13:47:51,312] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 13:47:51,399] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 13:47:51,400] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 13:47:51,401] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 13:47:51,403] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 13:47:51,403] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 13:47:51,768] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO Utils: Successfully started service 'sparkDriver' on port 38341.
[2021-12-28 13:47:51,810] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 13:47:51,867] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 13:47:51,908] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 13:47:51,909] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 13:47:51,916] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 13:47:51,947] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-bb7ac66e-addf-444d-b2d5-346572603b9c
[2021-12-28 13:47:51,973] {spark_submit.py:514} INFO - 21/12/28 13:47:51 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 13:47:52,000] {spark_submit.py:514} INFO - 21/12/28 13:47:52 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 13:47:52,292] {spark_submit.py:514} INFO - 21/12/28 13:47:52 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2021-12-28 13:47:52,386] {spark_submit.py:514} INFO - 21/12/28 13:47:52 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4040
[2021-12-28 13:47:52,690] {spark_submit.py:514} INFO - 21/12/28 13:47:52 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:47:52,780] {spark_submit.py:514} INFO - 21/12/28 13:47:52 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:47:52,782] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:47:52,783] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:47:52,785] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:47:52,786] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:47:52,787] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:47:52,790] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:47:52,791] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:47:52,793] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:47:52,794] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:47:52,795] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:47:52,795] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:47:52,796] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:47:52,796] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:47:52,797] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:47:52,798] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:47:52,798] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:47:52,799] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:47:52,800] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:47:52,800] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:47:52,801] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:47:52,801] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:47:52,802] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:47:52,803] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:47:52,803] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:47:52,804] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:47:52,804] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:47:52,805] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:47:52,806] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:47:52,806] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:47:52,807] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:47:52,808] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:47:52,808] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:47:52,809] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:47:52,810] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:47:52,810] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:47:52,811] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:47:52,811] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:47:52,812] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:47:52,813] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:47:52,813] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:47:52,814] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:47:52,815] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:47:52,815] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:47:52,816] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:47:52,817] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:47:52,817] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:47:52,818] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:47:52,818] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:47:52,819] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:47:52,820] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:47:52,820] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:47:52,821] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:47:52,822] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:47:52,822] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:47:52,823] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:47:52,823] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:47:52,824] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:48:12,653] {spark_submit.py:514} INFO - 21/12/28 13:48:12 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:48:12,672] {spark_submit.py:514} INFO - 21/12/28 13:48:12 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:48:12,673] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:48:12,674] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:48:12,675] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:48:12,676] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:48:12,677] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:48:12,678] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:48:12,679] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:48:12,680] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:48:12,681] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:48:12,683] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:48:12,684] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:48:12,685] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:48:12,686] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:48:12,686] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:48:12,687] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:48:12,688] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:48:12,689] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:48:12,689] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:48:12,690] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:48:12,691] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:48:12,692] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:48:12,693] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:48:12,694] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:48:12,695] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:48:12,696] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:48:12,697] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:48:12,698] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:48:12,699] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:48:12,700] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:48:12,701] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:48:12,702] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:48:12,703] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:48:12,704] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:48:12,704] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:48:12,705] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:48:12,706] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:48:12,706] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:48:12,707] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:48:12,707] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:48:12,708] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:48:12,709] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:48:12,710] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:48:12,711] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:48:12,712] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:48:12,712] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:48:12,713] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:48:12,714] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:48:12,714] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:48:12,715] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:48:12,716] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:48:12,716] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:48:12,717] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:48:12,718] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:48:12,718] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:48:12,719] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:48:12,720] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:48:12,720] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:48:32,654] {spark_submit.py:514} INFO - 21/12/28 13:48:32 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:48:32,718] {spark_submit.py:514} INFO - 21/12/28 13:48:32 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:48:32,719] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:48:32,721] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:48:32,723] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:48:32,724] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:48:32,725] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:48:32,726] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:48:32,728] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:48:32,729] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:48:32,730] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:48:32,731] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:48:32,732] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:48:32,733] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:48:32,743] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:48:32,744] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:48:32,745] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:48:32,746] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:48:32,747] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:48:32,748] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:48:32,752] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:48:32,754] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:48:32,755] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:48:32,756] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:48:32,757] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:48:32,758] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:48:32,759] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:48:32,760] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:48:32,761] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:48:32,762] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:48:32,763] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:48:32,763] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:48:32,764] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:48:32,765] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:48:32,766] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:48:32,767] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:48:32,769] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:48:32,769] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:48:32,770] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:48:32,771] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:48:32,772] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:48:32,773] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:48:32,774] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:48:32,775] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:48:32,776] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:48:32,777] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:48:32,779] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:48:32,780] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:48:32,781] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:48:32,782] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:48:32,783] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:48:32,784] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:48:32,785] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:48:32,786] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:48:32,787] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:48:32,788] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:48:32,789] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:48:32,790] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:48:32,791] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:48:52,620] {spark_submit.py:514} INFO - 21/12/28 13:48:52 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 13:48:52,626] {spark_submit.py:514} INFO - 21/12/28 13:48:52 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 13:48:52,641] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 44033.
[2021-12-28 13:48:52,642] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO NettyBlockTransferService: Server created on 0c125a5bba95:44033
[2021-12-28 13:48:52,644] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 13:48:52,647] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4040
[2021-12-28 13:48:52,659] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 13:48:52,661] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 44033, None)
[2021-12-28 13:48:52,665] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 13:48:52,674] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:44033 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 44033, None)
[2021-12-28 13:48:52,679] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 44033, None)
[2021-12-28 13:48:52,682] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 44033, None)
[2021-12-28 13:48:52,685] {spark_submit.py:514} INFO - 21/12/28 13:48:52 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 13:48:52,715] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 13:48:52,749] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO MemoryStore: MemoryStore cleared
[2021-12-28 13:48:52,750] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO BlockManager: BlockManager stopped
[2021-12-28 13:48:52,762] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 13:48:52,767] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 13:48:52,787] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 13:48:52,954] {spark_submit.py:514} INFO - 21/12/28 13:48:52 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 13:48:52,955] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:48:52,956] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:48:52,957] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:48:52,959] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:48:52,960] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:48:52,961] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:48:52,962] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:48:52,963] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:48:52,967] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:48:52,969] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:48:52,970] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:48:52,970] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:48:52,971] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:48:52,973] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:48:52,973] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:48:52,975] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:48:52,976] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:48:52,977] {spark_submit.py:514} INFO - 21/12/28 13:48:52 INFO SparkContext: SparkContext already stopped.
[2021-12-28 13:48:52,978] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 13:48:52,979] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-28 13:48:52,980] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-28 13:48:52,981] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 13:48:52,982] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 13:48:52,983] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 13:48:52,984] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 13:48:52,986] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 13:48:52,987] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 13:48:52,988] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 13:48:52,989] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 13:48:52,990] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:48:52,991] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:48:52,992] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:48:52,993] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:48:52,994] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:48:52,995] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:48:52,996] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:48:52,999] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:48:53,000] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:48:53,001] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:48:53,002] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:48:53,003] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:48:53,004] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:48:53,005] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:48:53,007] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:48:53,008] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:48:53,009] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:48:53,010] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:53,084] {spark_submit.py:514} INFO - 21/12/28 13:48:53 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 13:48:53,086] {spark_submit.py:514} INFO - 21/12/28 13:48:53 INFO ShutdownHookManager: Deleting directory /tmp/spark-166e2c6f-34ea-4a09-9db1-9cb400e3d2aa
[2021-12-28 13:48:53,091] {spark_submit.py:514} INFO - 21/12/28 13:48:53 INFO ShutdownHookManager: Deleting directory /tmp/spark-bc7144cd-e6a8-4eb8-a486-3e99ca2d846c
[2021-12-28 13:48:53,208] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py galway. Error code is: 1.
[2021-12-28 13:48:53,216] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=scraping_daft_galway, task_id=scrap_data, execution_date=20211227T093500, start_date=20211228T134747, end_date=20211228T134853
[2021-12-28 13:48:53,235] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/email.py:102: PendingDeprecationWarning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=smtp_mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2021-12-28 13:48:53,236] {email.py:208} INFO - Email alerting: attempt 1
[2021-12-28 13:48:53,710] {email.py:220} INFO - Sent an alert email to ['clement.liscoet@gmail.com']
[2021-12-28 13:48:54,715] {standard_task_runner.py:91} ERROR - Failed to execute job 5113 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py galway. Error code is: 1.
[2021-12-28 13:48:54,763] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 13:48:54,880] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
